packages/extrinsics/package.json

```diff
@@ -41,5 +41,6 @@
         "default": "./dist/index.js"
       }
     }
-  }
+  },
+  "dependencies": {}
```

Comment:
only if not intentional
```suggestion
```

Reply:
no i tried adding vitest everywhere to see if that fixed it